src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.lock.json

```diff
@@ -1878,33 +1878,7 @@
           "lib/dotnet/xunit.assert.dll": {}
         }
       },
-      "xunit.console.netcore/1.0.2-prerelease-00088": {
```

Comment:
Yes you should try and understand what happen here as I agree it doesn't look right.